src/AndroidDebugLauncher/AndroidDebugLauncher.csproj

```diff
@@ -34,6 +34,25 @@
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug.Desktop|AnyCPU'">
```

Comment:
I think you should have a Lab.Debug.Desktop Lab.Release.Desktop also. This is because there is a LAB flag that's set in builds\miengine.settings.targets which is used in preprocessor checks in both MIEngine and OpenDebugAD7